packages/store/test/test-encodeKey.js

```diff
@@ -114,3 +124,30 @@ test('order invariants', t => {
     }
   }
 });
+
+test('BigInt values round-trip', async t => {
+  await fc.assert(
+    fc.property(fc.bigInt(), n => {
+      const rt = decodeKey(encodeKey(n));
+      return assertionPassed(t.is(rt, n), () => rt === n);
```

Comment:
Is it important to return something from the 2nd arg of `fc.property`? Doesn't this suffice?

```suggestion
      t.is(rt, n);
```

Reply:
The examples at https://github.com/dubzzz/fast-check/#getting-started made it look like returning true(thy) was important, but https://dubzzz.github.io/fast-check/index.html#property suggests that only exactly `false` or an exception are treated as failures. I'll update accordingly.

Reply:
Hmm, the problem with skipping `return` is that doing so bypasses the fast-check behavior for presenting a reduced counterexample to the property being asserted—rather than something useful like
```
  encodeKey › BigInt encoding comparison corresponds with numeric comparison

  Rejected promise returned by test. Reason:

  Error {
    message: `Property failed after 6 tests␊
    { seed: 1977596688, path: "5:71:124", endOnFailure: true }␊
    Counterexample: [35n,-23836150654801173763396695620220553254412511975658213618617502216148950167540n]␊
    Shrunk 2 time(s)␊
    Got error: Property failed by returning false␊
    ␊
    Hint: Enable verbose mode in order to have the list of all failing values encountered during the run`,
  }
```
we just get
```
  encodeKey › BigInt encoding comparison corresponds with numeric comparison

  Difference:

  - true
  + false
```

I think I'd like to keep `assertionPassed` until it is made superfluous by upgrade to ava v4+ (in which assertions return a "passed" boolean, allowing replacement of `return assertionPassed(t.<method>(…), () => …)` with `return t.<method>(…)`).

Reply:
oh wow! Thanks! I was kinda disappointed that fast-check didn't seem to do the usual shrinking thing. Good to know I was just holding it wrong.